edx_proctoring/api.py

```diff
@@ -2507,32 +2508,32 @@ def _get_proctored_exam_view(exam, context, exam_id, user_id, course_id):
     elif attempt_status == ProctoredExamStudentAttemptStatus.timed_out:
         raise NotImplementedError('There is no defined rendering for ProctoredExamStudentAttemptStatus.timed_out!')
     elif attempt_status == ProctoredExamStudentAttemptStatus.submitted:
-        student_view_template = None if _was_review_status_acknowledged(
+        student_view_template = None if (_was_review_status_acknowledged(
             attempt['is_status_acknowledged'],
             exam
-        ) else 'proctored_exam/submitted.html'
+        ) and getattr(settings, 'PROCTORED_EXAM_VIEWABLE_PAST_DUE', False)) else 'proctored_exam/submitted.html'
```

Comment:
#NIT Can we please assign this variable to a variable in the constant.py? This is slightly more difficult to read?